runtime/src/accounts_db.rs

```diff
@@ -1144,6 +1144,8 @@ pub struct AccountsDb {
     // lower passes = faster total time, higher dynamic memory usage
     // passes=2 cuts dynamic memory usage in approximately half.
     pub num_hash_scan_passes: Option<usize>,
+
+    pub(crate) log_dead_slots: AtomicBool,
```

Comment:
is this part of this pr?

Reply:
Ah it is. Sorry - file was minimized in github so it didn't show the callers.
add /// desc

Reply:
add a `///` description to this field. `/// allow disabling noisy log`